rcl/test/rcl/test_subscription.cpp

```diff
@@ -213,7 +213,7 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
     ret = rcl_publish(&publisher, &msg);
     // TODO(wjwwood): re-enable this fini when ownership of the string is resolved.
     //                currently with Connext we will spuriously get a double free here.
-    // std_msgs__msg__String__fini(&msg);
+    std_msgs__msg__String__fini(&msg);
```

Comment:
I had forgotten about this. Did you have to do anything else to make this work (some other patch somewhere)?